lib/index.js

```diff
@@ -354,11 +354,11 @@ Segment.prototype.retrieveCrossDomainId = function(callback) {
     var writeKey = this.options.apiKey;
 
     // Exclude the current domain from the list of servers we're querying
-    var currentTld = getTld(window.location.hostname);
+    var currentTld = topDomain(window.location.hostname);
```

Comment:
we should probably add a requirement in our `package.json` rather than assume this function exists?